october-2017/17.md

```diff
@@ -1,20 +1,57 @@
+
+### Extend Site Moderation Queue
+
+#### Approval Process
+It's no secret that getting a modification approved on the MyBB website can be a slow process. The main offender for this is our willingness to make sure submissions are safe and of good quality, which means exploring the code and installing them for ourselves. This just isn't a sustainable practice and results in a lot of frustration.
+
+To speed up your addon submissions, we're going to be easing our policy for allowing submissions. Modifications will be placed into their own category when they're quickly submitted, listing them as "unapproved". We will still be able to manually go through and approve them, but it won't prevent you from having your mod listed on the site.
+
+If you find an issue with one of these unapproved addons, or even one that has been approved, please report it to us via the Private Inquiries forum so we can investigate properly.
+
+#### Paid Modification Support
+We are also taking a look into our support of paid modifications. In the past, these have been banned from being posted anywhere on the forum. Paid authors create some great things, so we'd like to embrace them more going forward. Our current solution is allowing paid modifications in the Requests/Services/Jobs forum, which you can find below. We will continue looking into ways to improve, so keep an eye out for that.
+
+[![MyBB R/S/J Forum](../resources/visit-forum-white.svg)](https://community.mybb.com/forum-190.html)
+
+### The future of 2.0
+After a lot of discussion, both internally, on the forum and in Discord, we've decided on the path that the MyBB software will be following leading up to MyBB 2. While we can all agree that MyBB needs to be more modern, rewriting it entirely from scratch is an enormous task and is taking much longer than we anticipated. Judging from the input from users, as well as discussion among staff, our best option moving forward will be to further bridge the 1.x series toward MyBB 2, while updating it to be more usable and modern.
+
+![MyBB Poll results](./17-screen.png)
 
-##Official Demo Site
-It has long been one of our short falls that we haven't got a demo site set up. However, this is going to change. We currently have developed a back end script for this to be possible. Users can create a demo and this will be active for roughly 72 hours and will then expire. We have of course disabled certain features for security reasons and we are also providing a few well known plugins that you can install and see what MyBB can really do! After our new MyBB.com design has been completed our designers are going to work on the front end for our demo site so stay tuned for further updates on this!
+We have not decided on any specifics, but have discussed some of the following options:
```

Comment:
> We have not decided upon any specifics regarding possible enhancements for the MyBB 1.x series, though some of the following possibilities have been discussed:
> 
> - Using the Twig templating engine to provide an improved templating experience, benefiting theme authors, plugin authors and the core.
> - Implementing support for the SASS styling system, allowing the use of variables and functions when crafting themes.
> - Providing a brand new response default theme with full mobile support for the modern web.
> - Many more user experience improvements, such as implementing common plugins into the core software.

Reply:
This comment still seems to stand @Eric-Jackson - not sure why it shows as outdated...